travis/publish.sh

```diff
@@ -76,13 +76,12 @@ print_project_version() {
   ./mvnw help:evaluate -N -Dexpression=project.version|sed -n '/^[0-9]/p'
 }
 
-is_release_commit() {
+cleanup_release_trigger() {
   project_version="$(print_project_version)"
   if [[ "$project_version" =~ ^[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$ ]]; then
```

Comment:
this will only be true when running off a release commit, which means we already succeeded and can cleanup. I tested this locally.